packages/angular-cli/tsconfig.json

```diff
@@ -25,9 +25,7 @@
     }
   },
   "exclude": [
-    "blueprints/*/files/**/*"
-  ],
-  "includes": [
-    "./custom-typings.d.ts"
+    "blueprints/*/files/**/*",
+    "**/*.spec.ts"
```

Comment:
I'm going to remove and put back. This tsconfig was copied from the one in ast-tools.